gradle/verification-metadata.xml

```diff
@@ -346,6 +356,11 @@
             <sha256 value="bc9d5020ffce81eca245c9148054600134a209b3d279d7be38fa4f2c07f7a505" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="com.fasterxml.jackson.dataformat" name="jackson-dataformat-smile" version="2.17.0">
+         <artifact name="jackson-dataformat-smile-2.17.0.jar">
+            <sha256 value="e6b2bf2a53d1f88ec02f1833da24c208aa299876e47d37f7920b1e83e214baab" origin="Generated by Gradle"/>
+         </artifact>
+      </component>
       <component group="com.fasterxml.jackson.dataformat" name="jackson-dataformat-xml" version="2.15.0">
```

Comment:
Do we need to remove this one too?

Reply:
2.15.0 still exists for other plugins/modules that use jackson, this only upgrades it for xcontent (which is independent).